WalletWasabi/Wallets/IWallet.cs

```diff
@@ -10,7 +10,8 @@ namespace WalletWasabi.Wallets;
 
 public interface IWallet
 {
-	string WalletName { get; }
+	string Name { get; }
```

Comment:
Don't change the existing interface!

Reply:
I just wanted to take care of @kiminuo's expectation.

> btw: It's not clear why we have on L14 WalletName and below we have just Id. I would expect either:
> 
> WalletName & WalletId, or
> Name & Id

Please decide either 
1. WalletId & WalletName 
or 
3. Id & Name.

Reply:
So I guess you have only one choice.

Reply:
Fixed